src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -200,6 +204,10 @@ namespace AppInstaller::CLI::Workflow
         case ManifestInstaller::InstallerTypeEnum::Msi:
         case ManifestInstaller::InstallerTypeEnum::Nullsoft:
         case ManifestInstaller::InstallerTypeEnum::Wix:
+            if (isUpdate && installer.UpdateBehavior == ManifestInstaller::UpdateBehaviorEnum::UninstallPrevious)
+            {
+                // TODO: hook up with uninstall when uninstall is implemented

```

Comment:
>// TODO: hook up with uninstall when uninstall is implemented [](start = 16, length = 61)

For now, this should probably terminate the context to prevent the install from happening.